### PR TITLE
PCPARTS- 15 | create Header and Footer components

### PR DESCRIPTION
**Pull request by monika-klokosinska-intive**
## PR Checklist

-   [x] Title format: **PCPARTS-15 | Title Case Description**
-   [x] **README** is up to date OR not necessary

## JIRA Ticket

[JIRA-XXX](https://exdorf.atlassian.net/browse/PCPARTS-15)

## Description

Introduces a header and footer to the page.

### CODE REVIEW

File: frontend/src/components/Header/index.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import styles from './header.module.css';
+
+function Header() {
+    return (
+    <header className={styles.header}>PCPARTS
+        <span className={styles.subtitle}>Select the best part for your PC!</span>
+    </header>)
```
Comment (IntiveMichal) [inline frontend/src/components/Header/index.tsx]: be consistent with the semicolon :)

**Review by IntiveMichal — APPROVED** [review]

File: frontend/src/components/Footer/footer.module.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.footer {
+    background-color: #007ac9;
+    color: hsla(0,0%,94%,.95);
```
Comment (IntiveMichal) [inline frontend/src/components/Footer/footer.module.css]: Please use one type of colors system

File: frontend/src/components/Header/header.module.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
+.header {
+    height: 80px;
+    color:  hsla(0,0%,94%,.95);
```
Comment (IntiveMichal) [inline frontend/src/components/Header/header.module.css]: Please use one type of colors system

File: frontend/src/components/Header/index.tsx (hunk 1 above)
Comment (monika-klokosinska-intive) [reply frontend/src/components/Header/index.tsx]: @IntiveMichal whoops! Added, thanks :).

File: frontend/src/components/Footer/footer.module.css (hunk 2 above)
Comment (monika-klokosinska-intive) [reply frontend/src/components/Footer/footer.module.css]: @IntiveMichal I switched to rgba everywhere :).

**Pull request merged**